web/src/search/helpers.tsx

```diff
@@ -74,6 +75,52 @@ export function toggleSearchFilter(query: string, searchFilter: string): string
     return (query.substring(0, idx).trim() + ' ' + query.substring(idx + searchFilter.length).trim()).trim()
 }
 
+export function getSearchTypeFromQuery(query: string): SEARCH_TYPES {
+    // RegExp to match `type:$TYPE` in any part of a query.
+    const matchSearchType = /(\b|^)type:\w*(\s*|$)/
+    const getTypeName = /(\b|^)type:(?<type>\w*)(\s|$)/
```

Comment:
`\b` includes the start of the string
```suggestion
    const getTypeName = /\btype:(?<type>\w*)(\s|$)/
```